### DISCUSSION TITLE
Replay shows slider break after break time, even though gameplay had no slider break

### DISCUSSION THREAD

**Discussion by iminlikewithyou** [ppy/osu#16670] [Q&A] [answered]
The slider break in question is at 1:50 in the song. As the title says, the slider break occurs in the replay, but it did not happen during gameplay, as this replay is a perfect score.

https://drive.google.com/file/d/1NZFzEBFxaU3XtVgZqaD1WnY-TyaiWBsc/view?usp=sharing

**Comment by peppy** [comment]
Was the replay set on stable or lazer?

**Comment by iminlikewithyou** [comment]
It's a stable replay

**Reply by peppy** [reply]
I'm not sure we can guarantee 100% accuracy, but we'll take a look into this.

**Comment by smoogipoo** [comment, marked as answer]
I'm not seeing any sliderbreak here. Just to confirm, you're talking about viewing the replay on lazer, correct? Can you show a video?

**Reply by iminlikewithyou** [reply]
Yep, here's the video
https://drive.google.com/file/d/1Zc8mv92F_GntpzBkVg4mHMiDTJq0S5DW/view?usp=sharing

**Reply by iminlikewithyou** [reply]
After seeing it again it's definitely not a slider break but more correctly an input that was registered too early from break time

**Reply by smoogipoo** [reply]
Yeah I'm not sure what's happening here. Your slider appears about 0.5-1 second earlier than mine. Do you have any universal offset perhaps? Maybe also try re-downloading the map?

**Reply by iminlikewithyou** [reply]
No offset - but redownloading the map fixed it 🤷🏻‍♂️
